commons/all/src/main/java/org/infinispan/commons/dataconversion/internal/Json.java

```diff
@@ -2891,8 +2867,7 @@ private void escapeJsonString(CharSequence plainText, Appendable out) throws IOE
       private boolean mustEscapeCharInJsString(int codepoint) {
          if (!Character.isSupplementaryCodePoint(codepoint)) {
```

Comment:
I am questioning this line here as well now.. I am thinking we can just replace this all with the `return c == '"' || c == '\\';` line

Reply:
It seems that is safe to do. I'll update and we see CI.

Reply:
CI seems ok.